client/my-sites/earn/memberships/index.jsx

```diff
@@ -480,16 +528,23 @@ class MembershipsSection extends Component {
 	}
 
 	renderOnboarding( cta ) {
+		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
 		return (
 			<div className="memberships__onboarding-wrapper">
 				<div className="memberships__onboarding-column-info">
 					<div className="memberships__onboarding-header">
-						{ this.props.translate( 'Introducing Recurring Payments.' ) }
+						{ isPayments
+							? this.props.translate( 'Introducing Payments.' )
+							: this.props.translate( 'Introducing Recurring Payments.' ) }
 					</div>
 					<p className="memberships__onboarding-paragraph">
-						{ this.props.translate(
-							'Start collecting subscription payments! Recurring Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to collect paying subscribers.'
-						) }{ ' ' }
+						{ isPayments
+							? this.props.translate(
+									'Start selling and collecting subscription payments! Payments is a feature inside the block editor. When editing a post or a page you can insert a button that will allow you to sell products or collect paying subscribers.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.